app/grandchallenge/reader_studies/views.py

```diff
@@ -939,7 +943,15 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ReaderStudyViewSet(ReadOnlyModelViewSet):
+class DeleteGroundTruthPermission(DjangoObjectPermissions):
```

Comment:
Why are you using change permissions for a delete view? Surely `delete_` should be used?

Reply:
I wasn't sure on this, my reasoning was that any editor for a reader study should be allowed to delete the ground truth for it, which is what I wanted to reflect in the permission check here.

Reply:
Yes, so give those permissions to the editors group, they probably already have it as they can already delete answers that belong to their reader study from any user.

Reply:
Currently reader study editors do not have delete permissions for answer objects, no one does actually. So we need to assign them for all existing answers, which will probably take some time. Shall I include a management command for that?

Reply:
Not super necessary, it's a quick thing we can do in the shell:

```python
for rs in ReaderStudy.objects.all():
  assign_perm("delete_answer", rs.editors.group, rs.answers.all())
```